pkg/api/grpcServer.go

```diff
@@ -83,6 +84,36 @@ func NewGRPCServer(aspectDispatcher adapterManager.AspectDispatcher, dispatcher
 	}
 }
 
+// compatBag implements compatibility between destination.* and target.* attributes.
+type compatBag struct {
+	parent attribute.Bag
+}
+
+// if a destination.* attribute is missing, check the corresponding target.* attribute.
```

Comment:
will this impact the "used attributes" calculation?  and do we care?

Reply:
It will not alter that calculation. It is checking for exactly one deprecated attribute in case the main one is not found.